app/src/main/java/com/nextcloud/talk/models/json/chat/ChatUtils.kt

```diff
@@ -31,7 +31,7 @@ class ChatUtils {
                 for (key in messageParameters.keys) {
                     val individualHashMap = messageParameters[key]
                     val type = individualHashMap?.get("type")
-                    if (type == "user" || type == "guest" || type == "call") {
+                    if (type == "user" || type == "guest" || type == "call" || type == "group" || type == "circle") {
                         resultMessage = resultMessage?.replace("{$key}", "@" + individualHashMap["name"])
```

Comment:
```suggestion
                    if (type == "user" || type == "guest" || type == "call") {
                        resultMessage = resultMessage?.replace("{$key}", "@" + individualHashMap["name"])
```

and instead add a `else` at the end:
```suggestion
                    } else {
                        resultMessage = resultMessage?.replace("{$key}", individualHashMap["name"])
```